src/components/forms/FormCheckbox.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @author psarando
+ */
+import React from "react";
+
+import getFormError from "./getFormError";
+
+import Checkbox from "@material-ui/core/Checkbox";
+import FormControl from "@material-ui/core/FormControl";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormHelperText from "@material-ui/core/FormHelperText";
+
+// Apparently only necessary for FastField, but maybe more correct for "vanilla" Field as well.
+const onCheckboxChange = (setFieldValue, fieldName) => (event, checked) => {
+    setFieldValue(fieldName, checked);
+};
```

Comment:
This does not look like the latest version of this file, since cyverse-de/ui-lib#51 updated this function.

Reply:
Oops, I think I forgot to do a pull 😩 .

Reply:
Ok. I doubled checked every file imported from ui-lib to make sure I have the latest version and updated the missed ones.